app/common/directives/ush-logo.html

```diff
@@ -0,0 +1,118 @@
+<a embed-control embed-only=true href="https://www.ushahidi.com" class="ushahidi-bug" target="_blank">
+  <svg class="ushahidi-logo" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 273.5 266.5" xml:space="preserve">
```

Comment:
We should already have the SVG for this in an include somewhere, we use it in the mode-bar